src/oca_pre_commit_hooks/checks_odoo_module_po.py

```diff
@@ -5,6 +5,7 @@
 from collections import defaultdict
 
 import polib
+from common import get_checks_docstring
```

Comment:
The `tests` folder is not part of the package itself

So, we can not import files from `tests` folder

We should move the code to package and from tests call the package method instead

Reply:
FYI I have already fixed